css/style.css

```diff
@@ -24,8 +24,52 @@ button a:hover{
   text-decoration: none;  
 }
 
+/*Stying for Tables made with div tags instead of table tags
+Adapted from: 
+http://www.dummies.com/web-design-development/css3/using-the-div-tag-to-create-tables/
+
+and from CSS and HTML generated at: 
+http://divtable.com/generator/
+*/
+.Table {
+    display: table;
+    
+}
+
+#recentReleasesTable{
+    display: inline-block;
+}
+
+.divTableBody{
+    display: table-row-group;
+}
+
+.Heading {
+    font-weight: bold;
+    text-align: center;
+    vertical-align: middle;
+    display: table-header-group;
+}
+
+.Heading .Cell{
+    border-bottom: 2px solid #000000;
+}
+
+.Row {
+    display: table-row;
+}
+
+.Cell {
+    display: table-cell;
+    text-align: center;
+    vertical-align: middle;
+    padding: 3px 10px;
+
+}
+
 table{ display: inline-block; }
 
+
```

Comment:
Will do. Just on a side note I will only be using style.min.css in the final release.